packages/razorsharp/src/ui/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FunctionComponent } from 'preact';
+import type { JSXInternal } from 'preact/src/jsx';
+import classNames from './Checkbox.module.css';
+
+export const Checkbox: FunctionComponent<{
```

Comment:
🙈 razorsharp 🤝 blade components?

Reply:
Had tried adding Blade but plugin size was getting out of hand. Eventually moved to Preact and just plain old CSS